.claude/commands/gap-planning.md

```
@@ -0,0 +1,500 @@
+## v2.0 Design Elements Reference
+
+### Colors
+```kotlin
+// Primary Gradient
+val PrimaryGradient = Brush.linearGradient(
+    colors = listOf(Color(0xFF667EEA), Color(0xFF764BA2))
+)
+
+// Secondary Gradient (Success)
+val SecondaryGradient = Brush.linearGradient(
+    colors = listOf(Color(0xFF11998E), Color(0xFF38EF7D))
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)